Dockerfile

```diff
@@ -1,18 +1,20 @@
 # Package path for this plugin module relative to the repo root
-ARG package=arcaflow_plugin_template_python
+ARG package=arcaflow_plugin_rtla
 
 # STAGE 1 -- Build module dependencies and run tests
 # The 'poetry' and 'coverage' modules are installed and verson-controlled in the
 # quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase image to limit drift
 FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.4.2 as build
 ARG package
 
+RUN dnf -y install rtla
+
 COPY poetry.lock /app/
 COPY pyproject.toml /app/
 
 # Convert the dependencies from poetry to a static requirements.txt file
-RUN python -m poetry install --without dev --no-root \
- && python -m poetry export -f requirements.txt --output requirements.txt --without-hashes
+RUN python3 -m poetry install --without dev --no-root \
+ && python3 -m poetry export -f requirements.txt --output requirements.txt --without-hashes
```

Comment:
Again leftover from experimentation; reverted.

I tried at one point using a fedora base image in order to get a newer rtla package, and I had to be explicit about python3 there because no alias existed.